frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -272,7 +272,7 @@ describe('Kubernetes resource CRUD operations', () => {
       await crudView.isLoaded();
       await crudView.resourceRows.$$('.co-kebab__button').click();
       await browser.wait(until.elementToBeClickable($('.co-kebab__dropdown')));
-      await element(by.cssContainingText('.co-kebab__dropdown a', 'View Instances')).click();
+      await element(by.cssContainingText('.co-kebab__dropdown button', 'View Instances')).click();
```

Comment:
Prefer this helper for kebab actions:

https://github.com/openshift/console/blob/master/frontend/integration-tests/views/crud.view.ts#L64-L68